dotCMS/src/main/java/com/dotmarketing/image/filter/HsbImageFilter.java

```diff
@@ -50,9 +50,9 @@ public File runFilter(File file,  Map<String, String[]> parameters) {
 			BufferedImage dst = filter.filter(src, null);
 			ImageIO.write(dst, "png", resultFile);
 			dst.flush();
-		} catch (IOException e) {
-			Logger.error(this.getClass(), e.getMessage());
-		}
+	    } catch (Exception e) {
+	        throw new DotRuntimeException("unable to convert file:" +file + " : " +  e.getMessage(),e);
```

Comment:
May be add a log and throw the exception?

Reply:
Probably it will logged in an upper level when it caught.